controllers/timechaos/types.go

```diff
@@ -285,11 +286,19 @@ func (r *Reconciler) applyContainer(ctx context.Context, client chaosdaemon.Chao
 		return err
 	}
 
-	r.Log.Info("setting time shift", "mask", mask, "sec", chaos.Spec.TimeOffset.Sec, "nsec", chaos.Spec.TimeOffset.NSec)
+	duration, err := time.ParseDuration(chaos.Spec.TimeOffset)
```

Comment:
split this logic to a single help function?  So we can add the unit test for this.

Reply:
Yes!